src/definitions.rs

```diff
@@ -19,7 +19,7 @@ pub enum State {
     #[default]
     Ground = 12,
     OscString = 13,
-    SosPmApcString = 14,
+    Null = 14,
```

Comment:
This is just a bad name. A "null state" already exists, it's called `Ground`. Looking at `State::Null` alone would make it impossible to tell what that state is for.